runbotics/runbotics-desktop/src/action/jira/jira.utils.ts

```diff
@@ -241,6 +247,59 @@ export const getJiraAllSprintTasks = async <T extends CloudJiraUser>(
     };
 };
 
+export const getJiraAllTaskDetails = async <T extends CloudJiraUser>(
+    input: GetTaskDetailsInput,
+) => {
+    const jql = `key="${input.task}"`.trim();
+
+    let startAt = 0;
+    let issues: SearchIssue<T> | null = null;
+
+    const response = await searchTaskDetails<T>(input, jql, startAt);
+    if (response.total > 0) {
+        issues = response.issues[0];
+    }
+
+    return {
+        issues,
+    };
+};
+
+const searchTaskDetails = async <T extends CloudJiraUser>(
+    input: GetSprintTasksInput,
+    jql: string,
+    startAt: Page['startAt'],
+)  => {
+    const { username, password, originUrl } = input;
+    const baseFields = 'summary, issuetype, created, resolutiondate, updated, status, assignee, issuelinks, priority, reporter, components, labels, sprint, epic link, timespent, duedate, customfield_10020, customfield_10014';
```

Comment:
that's a detail but I feel like there's a bigger probability to make a mistake with a string like that instead of an array. Also it's not being modified, so I'd make baseFields a const BASE_FIELDS = ['summary', 'issue_type', ...].join(', ')